src/lang/lwc_en.properties

```diff
@@ -598,3 +598,9 @@ display=Display
 prefix=%reset%
 
 lwc.admin.denied=%dark_red%You are not an LWC admin!
+
+protection.info=%dark_green%Protection ID%gray%: %reset%%id% (Type: %type%) \n\
```

Comment:
I'd name this something like protection.interact.info.formatted to be more consistent with the other preexisting keys.